tajo-core/src/main/resources/webapps/admin/query_executor.jsp

```diff
@@ -87,6 +88,11 @@ function runQuery() {
   } else if(Math.ceil(Number($("#sizeLimit").val())) > 0) {
     SIZE_LIMIT = Number($("#sizeLimit").val()) * 1024 * 1024;
   }
+  if(Math.ceil(Number($("#rowLimit").val())) >= 10) {
```

Comment:
IMO, we don't have to set the upper bound. It would be better to allow a very large value for rowLimit if users want.